src/PIL/Image.py

```diff
@@ -1771,6 +1771,7 @@ def rotate(self, angle, resample=NEAREST, expand=0, center=None,
         :param center: Optional center of rotation (a 2-tuple).  Origin is
            the upper left corner.  Default is the center of the image.
         :param translate: An optional post-rotate translation (a 2-tuple).
+        :param backgroundcolor: An optional color for the color outside the transformed image
```

Comment:
End with a full stop.

Reply:
Perhaps "rotated" instead of "transformed"? We don't need to care how the transform happened.